nav2_smac_planner/src/a_star.cpp

```diff
@@ -545,10 +545,14 @@ typename AStarAlgorithm<NodeT>::AnalyticExpansionNodes AStarAlgorithm<NodeT>::ge
     node->motion_table.state_space), s(node->motion_table.state_space);
   from[0] = node->pose.x;
   from[1] = node->pose.y;
-  from[2] = node->pose.theta * node->motion_table.bin_size;
+  //TODO: Bin size removed from motion table for state lattice
```

Comment:
Don't just comment it out -- that'll break every other planner :laughing: 

Instead do the following:
- Make a new function, getRadiansFromBin()
- Make an implementation for Hybrid using bin_size
- Make an implementation for 2D throwing an exception as invalid
- Make an implementation for Lattice by using the primitive headings vector

Reply:
See other functions in A* with template overrides for examples

Reply:
Done. 
The only place I didn't know how to purse was `angle = reals[2] / node->motion_table.bin_size;` inside of `getAnalyticPath`. 
Do we still want to conduct analytic expansion with the lattice planner? I vaguely remember discussing that it way be left out for the first pass.